packages/signature-controller/src/SignatureController.test.ts

```diff
@@ -928,4 +937,77 @@ describe('SignatureController', () => {
       });
     });
   });
+
+  describe('logging controller events', () => {
+    it('sends proposed sign log event after approval is shown', async () => {
+      await signatureController.newUnsignedTypedMessage(
+        messageParamsMock,
+        requestMock,
+        versionMock,
+        { parseJsonData: false },
+      );
+
+      expect(messengerMock.call).toHaveBeenNthCalledWith(
+        1,
+        'LoggingController:add',
+        {
+          type: LogType.EthSignLog,
+          data: {
+            signingMethod: SigningMethod.EthSignTypedData,
+            stage: SigningStage.Proposed,
+            signingData: expect.anything(),
```

Comment:
We can still use the `expect.any<>`  function but is it worth asserting the standard fields of the data for the sake of safety?

Reply:
Good call, it's worth asserting couple of fields, fixed in https://github.com/MetaMask/core/pull/1692/commits/f67bcbe6345a57cce5b94bf6401142340f9cf959